tests/python_api/audio_unit_test.py

```diff
@@ -80,7 +80,7 @@ def verify_output(audio_tensor, rocal_data_path, roi_tensor, test_results, case_
     for i in range(roi_data[0]):
         for j in range(roi_data[1]):
             ref_val = data_array[i * roi_data[1] + j]
-            out_val = audio_data[i * dimensions[2] + j]
+            out_val = audio_data[i * roi_data[1] + j]   # Stride upto max_roi
```

Comment:
we need to make roi as struct so it is clear what you are adding here

Reply:
This is the roi output from data-loader, hence storing in variables instead of structure